infra/repo_manager_test.py

```diff
@@ -133,5 +135,59 @@ def test_checkout_invalid_pull_request(self):
         test_repo_manager.checkout_pr('not/a/valid/pr')
 
 
+class GitDiffUnitTests(unittest.TestCase):
```

Comment:
Don't we normally name these singular, e.g. `GitDiffUnitTest` same for next class.

Reply:
Done.